src/FakeItEasy/Core/FakeManager.PropertyBehaviorRule.cs

```diff
@@ -39,25 +39,18 @@ public static bool IsPropertyGetter(MethodInfo method)
                 return method.IsSpecialName && method.Name.StartsWith("get_", StringComparison.Ordinal);
             }
 
+            public bool IsMatchForSetter(IFakeObjectCall fakeObjectCall) => IsPropertySetter(fakeObjectCall);

```

Comment:
> is the this still required (encouraged)?

Yes. Not sure why StyleCop doesn't detect it. Perhaps it's confused by the expression-bodied method.

> I'm not really sure why we need this method as well as IsPropertySetter

We don't really need it, but the `IsPropertySetter` instance method is private, and if I made it public, it would be strange if `IsPropertyGetter` were not also public, although it's not needed. Also, I thought `IsMatchForSetter` communicated the intent better.